WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -547,6 +547,12 @@ protected void onCreate(Bundle savedInstanceState) {
         setTitle(SiteUtils.getSiteNameOrHomeURL(mSite));
         mSectionsPagerAdapter = new SectionsPagerAdapter(fragmentManager);
 
+        setupViewPager();
```

Comment:
Hey @khaykov,
I moved the code into a separate method as our changes have exceeded the allowed number of lines per method which resulted in failed lint tests.